toolchain/sem_ir/formatter.cpp

```diff
@@ -447,6 +447,12 @@ class InstNamer {
       };
 
       switch (inst.kind()) {
+        case AddrPattern::Kind: {
+          // TODO: Visit the entire parameter block, not just the parameter
+          // refs, so that we visit the parameter inside an `addr` pattern.
```

Comment:
Is this an issue where patterns generally need to be recursed through? I'll offer some concern that walking the entire parameter block might not yield the right result either, due to separate use of blocks.

Reply:
Switched TODO to just describe the problem rather than suggesting a solution.